lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -399,7 +409,12 @@ def parse(body)
         JSON.parse(body)
       end
 
+      def endpoint(action)
+        action == 'disable' ? 'Recurring/' : 'Payment/'
+      end
+
       def commit(action, parameters, options)
+        url = url(action)
         begin
           raw_response = ssl_post("#{url}/#{action}", post_data(action, parameters), request_headers(options))
```

Comment:
Nitpick: Since we now pass in `action` to the `url` method, could we have it return the complete URL including the specific action endpoint, such that the line here reduces to,
```ruby
raw_response = ssl_post(url(action), post_data(action, parameters), request_headers(options))
```

Reply:
I like this -- thanks for the suggestion!